docs/helm-broker/docs/03-04-bundles-docs.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Bundles docs
+type: Details
+---
+
+## Documentation for ServiceClasses provided by bundle
+
+The Helm Broker in the provisioning process pushes the `addonRepositoryURL` variable into installed chart. With this feature a bundle which provides its own ServiceClasses e.g. `Azure Broker` can also install a documentation for them.
+
+If the bundle provides ServiceClasses and you want to have a documentation for it, you should create a `docs.yaml` inside the bundle's chart with the following structure:
+```
+apiVersion: cms.kyma-project.io/v1alpha1
+kind: ClusterDocsTopic
+metadata:
+  labels:
+    cms.kyma-project.io/view-context: service-catalog
+  name: {ServiceClass ID}
+spec:
+  displayName: "Slack Connector Add-on"
+  description: "Overall documentation, OpenAPI and AsyncAPI for Slack Connector Add-on"
+  sources:
+    - type: {type}
+      name: {name}  # must be unique within sources
+      mode: {mode}
+      url: {.Values.addonsRepositoryURL}
+      filter: docs/{class_name}/   # provide filter only if you use the package mode
+___
+# it is prefered to store all of the ClusterDocsTopics definitions in one file
```

Comment:
this sentence: "It is preferred" should be moved from the example to the normal documentation paragraph

Reply:
done